src/main.c

```diff
@@ -45,11 +45,6 @@ static void signal_handler(int sig) {
 int main(int argc, char *argv[]) {
     if (detect_sandbox()) return EXIT_SUCCESS;
 
-    if (geteuid() != 0) {
-        fail("FATAL: "APP_NAME" does not have elevated privileges required for mount operations.\n"
-             "Executable should be owned by root with setuid bit enabed.\n")
-    }
-
```

Comment:
The program needs to fail if it does not have the correct permissions set. This should be replaced with checking if CAP_SYS_ADMIN is set.